### ISSUE TITLE
Invalid array length

### ISSUE THREAD

**Issue by inhydrox** [DevYukine/Kurasuta#398]
```js
(node:776) UnhandledPromiseRejectionWarning: RangeError: Invalid array length
    at ShardingManager.spawn (/home/runner/tuneybot/node_modules/kurasuta/dist/Sharding/ShardingManager.js:71:36)
    at /home/runner/tuneybot/index.js:18:9
    at Script.runInContext (vm.js:130:18)
    at Object.<anonymous> (/run_dir/interp.js:209:20)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
(node:776) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:776) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

**Comment by inhydrox** [comment]
Code: ```js
const sharder = new ShardingManager(__dirname + "/app.js", {
  token: config.token,
  client: Client,
  respawn: true,
  retry: true,
  ipcSocket: 9999,
});
```

**Comment by TornadoMC2** [comment]
> Code:
> 
> ```js
> const sharder = new ShardingManager(__dirname + "/app.js", {
>   token: config.token,
>   client: Client,
>   respawn: true,
>   retry: true,
>   ipcSocket: 9999,
> });
> ```

try removing the ipcSocket value since its not needed and try again

**Comment by DevYukine** [comment]
fixed in cecaedfa0104acce62eb5f38c4f541e1c7d95601
